pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -675,21 +784,28 @@ func (nc *Controller) doEvictionPass() {
 				klog.Warningf("Failed to get Node %v from the nodeLister: %v", value.Value, err)
 			}
 			nodeUID, _ := value.UID.(string)
-			pods, err := listPodsFromNode(nc.kubeClient, value.Value)
-			if err != nil {
-				utilruntime.HandleError(fmt.Errorf("unable to list pods from node %q: %v", value.Value, err))
+			pods, ok := nc.nodeEvictionMap.deletePods(value.Value)
+			if !ok {
+				utilruntime.HandleError(fmt.Errorf("node %q eviction data doesn't exist", value.Value))
```

Comment:
This isn't an error per-se - it may happen in a healthy system and it is fine.

It requires a comment describing when it can happen and why it is fine to ignore it.

Reply:
Done.